src/assets/stylesheets/index.scss

```diff
@@ -356,3 +356,45 @@ body {
   }
 }
 
+#mc_embed_signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
This form is cut off on mobile screens:
![image](https://user-images.githubusercontent.com/79419/78593797-0d7d4980-77fc-11ea-855a-27dc3516ec8d.png)